Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XLGToDBAnalyzer.cs

```diff
@@ -162,7 +182,8 @@ public XLGToDBAnalyzerInner(AnalysisInput input) : base(input)
         /// <inheritdoc/>
         public override void Prepare()
         {
-            var accessor = KeyValueStoreAccessor.Open(storeDirectory: OutputDirPath);
+            string[] additionalColumns = { XldbDataStore.EventColumnFamilyName, XldbDataStore.PipColumnFamilyName, XldbDataStore.StaticGraphColumnFamilyName };
+            var accessor = KeyValueStoreAccessor.Open(storeDirectory: OutputDirPath, additionalColumns: additionalColumns);

```

Comment:
In this use-case, where you're only writing in bulk to the DB, you may benefit from PrepareForBulkLoad(). See the [RocksDb FAQ](https://github.com/facebook/rocksdb/wiki/RocksDB-FAQ)

Reply:
Will look into this for a separate "OptimizeDB" pr :) 

---
In reply to: [314403390](https://github.com/microsoft/BuildXL/pull/724#discussion_r314403390) [](ancestors = 314403390)